### DISCUSSION TITLE
part-ii/2012-62/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#187] [Question comments]
Part II 2012 Galois Theory: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2012-62/

**Comment by eha1234** [comment]
<details><summary>First part</summary>

Conjugation is an automorphism of $\mathbb{C}$ with order 2. Now obtain a contradiction for $|G|$ odd. 
</details>

<details><summary>Second part</summary>

Take $K = \mathbb{Q}$. Find an irred cubic with three roots in $\mathbb{R}$ whose discriminant is square in $\mathbb{Q}$. Showing that this works is essentially bookwork, finding an example is a bit annoying. I took $x^2(x-1)+a$ and fiddled around with discriminant formula to get an appropriate $a$. The second example is easier to find - e.g. adjoin $\sqrt[3]{2} e^{2\pi i/3}$
</details>
